db/schema.rb

```diff
@@ -11,20 +11,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160523234717) do
+ActiveRecord::Schema.define(version: 20160606202534) do
 
   create_table "builds", force: :cascade do |t|
-    t.integer  "project_id",                       null: false
+    t.integer  "project_id",          limit: 4,    null: false
     t.integer  "number",              limit: 4
     t.string   "git_sha",             limit: 255
     t.string   "git_ref",             limit: 255
     t.string   "docker_image_id",     limit: 255
     t.string   "docker_ref",          limit: 255
     t.string   "docker_repo_digest",  limit: 255
-    t.integer  "docker_build_job_id"
+    t.integer  "docker_build_job_id", limit: 4
     t.string   "label",               limit: 255
     t.string   "description",         limit: 1024
-    t.integer  "created_by"
+    t.integer  "created_by",          limit: 4
     t.datetime "created_at"
     t.datetime "updated_at"
   end
```

Comment:
Hey @jonmoter, do you want me to remove all those **limit** changes added to the `db/schema.rb` file as well?